tests/app/views/test_suppliers.py

```diff
@@ -1280,13 +1280,23 @@ def setup(self):
                 data=json.dumps({'updated_by': 'interested@example.com'}),
                 content_type='application/json')
 
+            # framework_agreement_version gets saved into the agreementDetails JSON blob of a supplier_framework
+            # when an agreement is returned
+            Framework.query.filter_by(slug='digital-outcomes-and-specialists').update(
```

Comment:
We don't necessarily need a full copy of all the tests for different frameworks, but anything using the framework details field should have a test both with and without the details existing.